### PR TITLE
Map class draft

### PR DESCRIPTION
**Pull request by jasminekp**
Commit following files:
FIleManager.h (test header)
Map.cpp
Map_Reduce.cpp (generated by VS)
Test1.cpp
& general visual studio files

### CODE REVIEW

File: Map.cpp [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+#include <string>
+#include <vector>
+#include <list>
+#include <utility>
+#include <sstream>
+#include <cctype>
+#include "FileManager.h"
+using namespace std;
+
+class Map
```
Comment (jasminekp) [inline Map.cpp:11]: *Need to create a separate Map.h file for the next phase

File: Test1.cpp [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#define BOOST_TEST_MODULE mytests
+#include <boost/test/included/unit_test.hpp>
+
+BOOST_AUTO_TEST_CASE(myTestCase)
```
Comment (jasminekp) [inline Test1.cpp:4]: *To be updated further with actual test cases for Map.cpp

File: Map.cpp [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+		}
+	}
+
+	bool map(string fileName, std::string value)
+	{
+		// read in the string from the file through the FileManager class
+		string rawData = fm.readFile();
+
+		// tokenize the string into distinct words and remove punctuation, strings, etc
+		if (TokenizeValue(value))
```
Comment (jasminekp) [inline Map.cpp:73]: This method does not handle end of line characters just yet --> need to account for '\n' at the end of the line

**Review by jasminekp — COMMENTED** [review]
Added comments for this initial draft of the MapTest branch that will be revised in the next phase

File: FileManager.h [hunk 4]
```diff
@@ -0,0 +1,13 @@
+
+//sample setup for FileManager header file
+
+using namespace std;
```
Comment (risktakingkev) [inline FileManager.h:4]: It's good practice to explicitly use std:: throughout ones program. I suggest you do so.

File: Map.cpp [hunk 5]
```diff
@@ -0,0 +1,85 @@
+#include <iostream>
+#include <string>
+#include <vector>
+#include <list>
+#include <utility>
+#include <sstream>
+#include <cctype>
+#include "FileManager.h"
+using namespace std;
```
Comment (risktakingkev) [inline Map.cpp:9]: It's good practice to explicitly use std:: throughout ones program. I suggest you do so.

**Pull request closed without merging**